### PR TITLE
another commit

### PR DESCRIPTION
**Pull request by danthegoodman1**

### CODE REVIEW

File: balena.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 type: service-source@1.0.0
```
Comment (Hades32) [inline balena.yml]: You could change that to our new type

**Comment by danthegoodman1** [comment]
@balena-ci I self cerify!

**Comment by ghost** [comment]
Here's a list of commands supported by VersionBot
- `@balena-ci I self-certify!` - use [the self-certification flow](https://docs.google.com/drawings/d/1aoXCdi8bzinxaLM0BoKMJaiZjIJj5Az02Npw_L2iQkE/edit?usp=sharing) for this PR
- `@balena-ci test` - trigger the tests for this PR
- `@balena-ci version` - generate the versioned branch for your PR
- `@balena-ci rebase` - rebase the PR

**Comment by danthegoodman1** [comment]
@balena-ci I self certify!

**Pull request merged**